components/automate-ui/src/app/entities/environments/environment.model.ts

```diff
@@ -13,7 +13,12 @@ export interface Environment {
 
 export interface CookbookVersion {
   name: string;
-  version: string;
+}
+
+export interface CookbookVersionDisplay {
+  name?: string;
+  operator?: string;
+  versionNumber?: string;
```

Comment:
No need for these to be optional, is there? (Always prefer non-optional if you can)
```suggestion
  name: string;
  operator: string;
  versionNumber: string;
```

Reply:
updated :)